mobile/src/main/java/com/google/samples/apps/iosched/ui/MainActivity.kt

```diff
@@ -46,6 +46,7 @@ import com.google.samples.apps.iosched.shared.di.CodelabsEnabledFlag
 import com.google.samples.apps.iosched.shared.di.ExploreArEnabledFlag
 import com.google.samples.apps.iosched.shared.di.MapFeatureEnabledFlag
 import com.google.samples.apps.iosched.shared.domain.ar.ArConstants
+import com.google.samples.apps.iosched.shared.result.Event
```

Comment:
Is this used?

Reply:
Nope, I am surprised the CI checks didn't catch this. I thought I had organized imports before committing but evidently I didn't. 😅